packages/fxa-auth-server/test/local/routes/session.js

```diff
@@ -56,6 +79,24 @@ function hexString(bytes) {
   return crypto.randomBytes(bytes).toString('hex');
 }
 
+function getExpectedOtpCode(options = {}, secret = 'abcdef') {
+  const authenticator = new otplib.authenticator.Authenticator();
+  authenticator.options = Object.assign(otplib.authenticator.options, options, {
+    secret,
+  });
+  return authenticator.generate();
+}
+
+async function assertFailure(asyncFn, assertFn) {
+  let err;
+  try {
+    await asyncFn;
+  } catch (e) {
+    err = e;
+  }
+  assertFn(err);
+}
```

Comment:
And if you do decide to create `test/assert.js`, maybe you could put this in there too, I noticed it's defined again further down.